### PR TITLE
docs: JK-10274: Updated API link and created documentation card for i…

### PR DESCRIPTION
**Pull request by AedinC**
…ac tools guide.

### CODE REVIEW

File: .build/external-documentation/rhoas-iac-tools.quickstart.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  description: Learn how to use different infrastructure as code (IaC) tools to provision and manage Kafka instances in OpenShift Streams for Apache Kafka.
```
Comment (jbyrne-redhat) [inline .build/external-documentation/rhoas-iac-tools.quickstart.yml:15]: From what I can see in the console, the description  gets truncated after 117 characters. If you don't want that, something like the following might work:

"Learn how to use infrastructure as code (IaC) tools to create Kafka instances in OpenShift Streams for Apache Kafka."

File: docs/rhoas/iac_tools_rhoas/README.adoc [hunk 2]
```diff
@@ -93,7 +93,7 @@ As a system administrator of applications and services, you can use Infrastructu
 This guide describes how to use different IaC tools to provision and manage Kafka instances in {product-long-kafka}. To interact with Kafka instances, the IaC tools use two REST APIs that are available in {product-kafka} - a _management_ API and an _instance_ API.
 
 ifndef::community[]
-For more information about these APIs, see https://access.redhat.com/documentation/en-us/red_hat_openshift_application_services/1/guide/2409253a-45ee-470e-bdc9-5db4bfcf9d0f.
+For more information about these APIs, see https://access.redhat.com/documentation/en-us/red_hat_openshift_application_services/1/guide/2409253a-45ee-470e-bdc9-5db4bfcf9d0f[Interacting with Red Hat OpenShift Application Services using APIs].
```
Comment (jbyrne-redhat) [inline docs/rhoas/iac_tools_rhoas/README.adoc:96]: Include a caret (` ^`)  after the link title:

```
For more information about these APIs, see https://access.redhat.com/documentation/en-us/red_hat_openshift_application_services/1/guide/2409253a-45ee-470e-bdc9-5db4bfcf9d0f[Interacting with Red Hat OpenShift Application Services using APIs^]
```

File: .build/external-documentation/rhoas-iac-tools.quickstart.yml (hunk 1 above)
Comment (AedinC) [reply .build/external-documentation/rhoas-iac-tools.quickstart.yml:15]: Thanks John!

Comment (jbyrne-redhat) [reply .build/external-documentation/rhoas-iac-tools.quickstart.yml]: Your call on how (or whether) you want to update this description. It's a pity when the description gets truncated, but not the end of he world if the main gist of it is clear. My wording above is only a suggestion.  I know it leaves out the notion of _managing_ instances, for instance.

Comment (jbyrne-redhat) [reply .build/external-documentation/rhoas-iac-tools.quickstart.yml]: Another possibility:

"Learn how to use Ansible and Terraform to create and manage Kafka instances in OpenShift Streams for Apache Kafka."

**Review by jbyrne-redhat — APPROVED** [review]
LGTM!

**Pull request merged**